images/build/ruby/2.6/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Build ruby 2.6
+FROM ruby-build-prereqs AS ruby26-build
```

Comment:
we might not need these 'build' related dockerfiles since AppService might be interested in installing them dynamically. We could however install it in vso image but anyway these docker files should not be required. Also did you check if our build image should also have any pre-reqs installed for ruby?